sig-scalability/charter.md

```diff
@@ -0,0 +1,192 @@
+  <td>Subproject</td>
+  <td>Description</td>
+  <td>Example Artifacts</td>
+</tr>
+<tr>
+  <td>Kubernetes scalability</td>
+  <td>Defining what does it mean that “Kubernetes scales”. This includes defining
+  (or approving) individual performance SLIs/SLOs, ensuring they are all oriented
+  on user experience and consistent with each other.</td>
+  <td>API-machinery performance SLIs/SLOs</td>
+</tr>
+<tr>
+  <td>Kubernetes performance validation</td>
+  <td>Ensuring that each official Kubernetes release satisfies all scalability and
+  performance related requirements, as state in “Kubernetes scalability”
+  definition.</td>
+  <td>1.9 validation report</td>
+</tr>
+<tr>
+  <td>Scalability testing frameworks</td>
+  <td>Designing and creating frameworks to make scalability and performance testing
+  of Kubernetes easy and available for all contributors. Different frameworks may
+  help in different aspect of scalability testing enabling making conscious tradeoffs,
+  e.g. cost of accuracy or real life vs more generalized benchmarking scenarios. </td>
+  <td>Cluster loader</td>
+</tr>
+<tr>
+  <td>Scalability and performanc tests</td>
+  <td>Ensuring that all tests necessary to validate Kubernetes scalability and
+  performance exist (ideally by providing easy-to-use framework and working with SIGs
+  to provide them) have the environment and resources to run on and are being
+  executed according to calendar enabling release validation.</td>
+  <td>Scalability e2e tests</td>
+</tr>
+<tr>
+  <td>Scalability governance</td>
+  <td>Establishing and documenting best practises on how to design and/or implement
+  Kubernetes features in scalable and performant way. Educating contributors and
+  ensuring those are widely used.</td>
+  <td>Regressions case study</td>
+</tr>
+</table>
+
+TODO: Figure out if we need subproject for finding bottlenecks, coordinating
+improvements and architectural changes, etc.
+
+[API-machinery performance SLIs/SLOs]: https://github.com/kubernetes/community/blob/master/sig-scalability/slis/apimachinery_slis.md
+[1.9 validation report]: https://github.com/kubernetes/sig-release/blob/master/releases/release-1.9/scalability_validation_report.md
+[Cluster loader]: https://github.com/kubernetes/perf-tests/tree/master/clusterloader
+[Scalability e2e tests]: https://github.com/kubernetes/kubernetes/tree/master/test/e2e/scalability
+[Regressions case study]: https://github.com/kubernetes/community/blob/master/sig-scalability/blogs/scalability-regressions-case-studies.md
+
+## Roles
+The following roles are required for the SIG to properly function.
+In the event that any role is unfilled, the SIG will make a best effort
+to fill it and any decisions reliant on a missing role will be postponed
+until the role is filled.
+
+### Chair
+- Number: 2-3
+- Run operations and processes governing the SIG
+- A majority of chairs cannot be from a single company.
+- An initial set of chairs was established at the time the SIG was founded.
```

Comment:
Who are the initial chairs?

Reply:
Bob Wise and myself

Reply:
Can you put it into the charter given that you have put TLs' names?

Reply:
done